content/en/security/ai_guard/onboarding.md

```diff
@@ -0,0 +1,540 @@
+---
+title: Get Started with AI Guard
+private: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/llm-guardrails-best-practices/"
+  tag: "Blog"
+  text: "LLM guardrails: Best practices for deploying LLM apps securely"
+---
+
+{{< site-region region="gov" >}}<div class="alert alert-danger">AI Guard isn't available in the {{< region-param key="dd_site_name" >}} site.</div>
+{{< /site-region >}}
+
+AI Guard helps secure your AI apps and agents in real time against prompt injection, jailbreaking, tool misuse, and sensitive data exfiltration attacks. This page describes how to set it up so you can keep your data secure against these AI-based threats.
+
+## Setup
+
+### Prerequisites
+
+Before you set up AI Guard, ensure you have everything you need:
+- While AI Guard is in Preview, Datadog needs to enable a backend feature flag for each organization in the Preview. Contact [Datadog support][1] with one or more Datadog organization IDs (or organization tenant names) to enable it.
```

Comment:
```suggestion
- While AI Guard is in Preview, Datadog needs to enable a backend feature flag for each organization in the Preview. Contact [Datadog support][1] with one or more Datadog organization names and regions to enable it.
```

Tweaked this, since users actually do not know about the concept of organization ID, it's usually confusing to them.